### PR TITLE
[TASK] Configure api via environment, server ejs

### PR DESCRIPTION
**Pull request by stevenzeiler**
Since configuration of the app should be performed via environment variables for consistency with the 12-factor application architecture, this pull request injects the url to the appropriate validator-registry-api as a server-rendered variable into the angular.js front end.

In the future we may have a build step that takes configuration from the environment and outputs hard-coded static files that we can serve from Amazon Cloud Front (S3).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,10 @@ npm install
 npm start
 ````
 
+## Configuration
+
+All configuration is injected with environment variables:
+
+PORT                   (default 1337)
+VALIDATOR_REGISTRY_API (default https://api.validators.ripple.com)
```
Comment (wilsonianb) [inline README.md:25]: Is there a case for not using this default?

Comment (stevenzeiler) [inline README.md:25]: Yeah in development or staging we need to use a different URL. Also if someone wants to point the webapp at their own registry.

Comment (wilsonianb) [inline README.md:25]: Sorry, I meant: are we sure we want to set `https://api.validators.ripple.com` as the default?

Comment (wilsonianb) [inline README.md:25]: Are we ok with people cloning this repo and hitting our server?

Comment (stevenzeiler) [inline README.md:25]: Yes.

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**